Casks/pdk.rb

```diff
@@ -2,16 +2,16 @@
   case MacOS.version
   when '10.11'
     os_ver = '10.11'
-    version '1.8.0.0'
-    sha256 'a5c404e7a33ca2f47aa575958a9f0f879ef5f5ac909e48fe2b6b0cfe69ff4fa7'
+    version '1.9.0.0'
+    sha256 '997ad11c3e6b0c300ff2a0b822db2e93eea28f0a19036b404f4b14451c84cb9c'
   when '10.12'
     os_ver = '10.12'
-    version '1.8.0.0'
-    sha256 '26548072664f6126975ba205e5334c28a05da68a50bd1534b4785c7293e196bc'
+    version '1.9.0.0'
+    sha256 '9da5599e376b33a4086d52f38e49ab1eb8c746cb4b6fe313251b303d2a3aca1e'
   else
     os_ver = '10.13'
-    version '1.8.0.0'
-    sha256 'f0418c82df0745c0968def1ec2639610777c7492bc979a4fd34c0a3dc33ecda1'
+    version '1.9.0.0'
+    sha256 '776fc996b55ee6e829ac88c48e3104b540f92d140d7c272a634104a896ef9db1'
```

Comment:
Did you generate this with the rake task? Looks like it didnt pick up 10.14?

Reply:
Oh, looks like there's an array we'll have to update it https://github.com/puppetlabs/homebrew-puppet/blob/master/Rakefile#L23.

Reply:
Updated rake task with #56 which outputs:
```
cask 'pdk' do
  case MacOS.version
  when '10.11'
    os_ver = '10.11'
    version '1.9.0.0'
    sha256 '997ad11c3e6b0c300ff2a0b822db2e93eea28f0a19036b404f4b14451c84cb9c'
  when '10.12'
    os_ver = '10.12'
    version '1.9.0.0'
    sha256 '9da5599e376b33a4086d52f38e49ab1eb8c746cb4b6fe313251b303d2a3aca1e'
  when '10.13'
    os_ver = '10.13'
    version '1.9.0.0'
    sha256 '776fc996b55ee6e829ac88c48e3104b540f92d140d7c272a634104a896ef9db1'
  else
    os_ver = '10.14'
    version '1.9.0.0'
    sha256 '509159873142378b282f8e6c1d645eb37480ecdc811747303057dbee77a95afc'
  end

  depends_on macos: '>= 10.11'
  url "https://downloads.puppet.com/mac/puppet5/#{os_ver}/x86_64/pdk-#{version}-1.osx#{os_ver}.dmg"
  pkg "pdk-#{version}-1-installer.pkg"

  name 'Puppet Development Kit'
  homepage 'https://github.com/puppetlabs/pdk'

  uninstall pkgutil: 'com.puppetlabs.pdk'
end
```

Reply:
Yeah, I was going to follow up with another PR to add 10.14 package once those are mirrored.